File: Data Structures & Algorithms/first-missing-positive/submission-2.go
func firstMissingPositive(nums []int) int {
	freq := make([]int,100001)
	min := 999999999999999
	for _, val := range nums {
		if val > 0 {
			if val < min {
				min = val
			}
			freq[val]++
		}
	}

	for i := 1; i < 100001;i++ {
		print(i)
		if freq[i] == 0 {
			return i
		}
	}
	return 1
}

File: Data Structures & Algorithms/monotonic-array/submission-2.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        
        isIncrease = 2
        for i in range(1,len(nums)):
            if nums[i] > nums[i - 1] and (isIncrease == 2 or isIncrease == 1):
                isIncrease = 1
            elif nums[i] < nums[i - 1] and (isIncrease == 2 or isIncrease == 0):
                isIncrease = 0
            elif nums[i] == nums[i - 1]:
                pass
            else:
                return False

        return True